src/ios/THEPeerBluetooth.m

```diff
@@ -213,7 +213,7 @@ @implementation THEPeerBluetooth
 
 // Class initializer.
 - (instancetype)initWithServiceType:(NSUUID *)serviceType
-                     peerIdentifier:(NSUUID *)peerIdentifier
+                     peerIdentifier:(NSString *)peerIdentifier
                            peerName:(NSString *)peerName
 {
     // Initialize superclass.
```

Comment:
At line 229 there is a check for peerName length. I realize this code predates you but you own it now. :) First, no magic numbers please. Let's get that length as a constant somewhere useful. Second of all truncating is bad. We depend on the peerName to communicate information and having that information disappear from under us wouldn't be good. Whatever the length limit is we need to return an error if it is violated. Please work with Jukka and Matt to agree on an error message and have Jukka put it into the Java code and Matt create a test for it and update our API doc to specify it.